src/components/RedirectionNotificationModal.js

```diff
@@ -43,47 +46,54 @@ export default function RedirectionNotificationModal({
     setSelectedDoNotShowRedirectionModal_Checkbox(false); //to avoid prechecked checkboxes
```

Comment:
Is "selected" necessary as part of the variable name? It has both checkbox and selected... maybe one is enough? Or something like: `doNotShowNotification` would suffice?

I might be tired at this time, but these very long names are hard to follow :)